packages/website/components/contexts/plansContext.js

```diff
@@ -1,27 +1,60 @@
-// temporary placeholder for stripe API data
-export const plans = [
+export const sharedPlans = [
```

Comment:
@cmunns I think I might need to make this driven by the server and not just the ui code.

e.g. on pageload the ui would need to do `GET /billing/plans/` and get back this information before rendering.

That way I can make sure it corresponds to prices/products in stripe that it knows how to bill for